### ISSUE TITLE
Incorrect node version used in GitPod

### ISSUE THREAD

**Issue by Amorpheuz** [forem/forem#8849]
**Describe the bug**
Creating a new GitPod instance leads to a bug where under `gem install solargraph; bin/setup && bin/startup` the `bin/setup` script fails at `yarn install` due to incorrect node version being used (which is 12.18.x). 

The [GitPod Dockerfile](https://github.com/thepracticaldev/dev.to/blob/master/.gitpod.dockerfile) does handle the installation of nvm and correct node version, but forgets to switch to the correct node version (12.16.3).

**To Reproduce**
1. Create a GitPod Workspace with https://gitpod.io/#https://github.com/thepracticaldev/dev.to
2. Wait till `bundle install` finishes in the third `workspace/dev.to` tab completes.
3. Yarn throws the error as indicated in screenshot below.

**Expected behavior**
Correct node version is selected in nvm, which allows the `yarn install` command to complete successfully.

**Screenshots**
![image](https://user-images.githubusercontent.com/22113778/85383294-6b990d80-b55d-11ea-92ff-a74d807c917b.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Edge
- Version: 83

**Additional context**
 The `nvm use` command can be of help.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by jacobherrington** [comment]
It looks like we might need to add `&& nvm use $NODE_VERSION` to https://github.com/thepracticaldev/dev.to/blob/2a31f7b9de7e9090ec908dd2a2d28508d807ab6f/.gitpod.dockerfile#L10

**Comment by citizen428** [comment]
I have a draft PR open for this but never quite got it working. Let me try again.

**Comment by akhil-naidu** [comment]
For now, you can manually change the version in package.json file. 

I'm not an expert to know the crux of what it actually does, but it gave me an active instance I'm my case.
